library/analytics/src/main/java/org/cru/godtools/analytics/AdobeAnalyticsService.java

```diff
@@ -202,6 +211,55 @@ private void trackState(@NonNull final String screen, @Nullable final Locale con
         if (contentLocale != null) {
             data.put(KEY_CONTENT_LANGUAGE, toLanguageTag(contentLocale));
         }
+
+        switch (screen) {
+            case "Home":
```

Comment:
use the screen constants for this switch, that way if they are ever updated the changes propagate here.